File: Codigo_Fonte_pascal/lista1/lista1_11.pas
program volume_esfera;

var
   raio, pi, volume, cubo : real;

begin
     pi := 3.14;
     write('Digite o raio da esfera :');
     readln(raio);

     cubo := sqr(raio) * raio;
     volume := (4 / 3) * pi * cubo;
     writeln('O volume da esfera = ', volume:0:2);

     readln;

end.

File: Codigo_Fonte_c/Lista 1/lista1_11.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(){
	float raio, pi, volume;
	pi = 3.14;
	printf("Digite o raio: ");
	scanf("%f", &raio);
	volume = 1.333333333 * pi * pow(raio, 3);

	printf("O volume da esfera de raio %.2f é igual a %.2f\n", raio, volume);

	return 0;
}
